dapps/templates/demo/config/blockchain.js

```diff
@@ -4,7 +4,8 @@ module.exports = {
   // default applies to all environments
   default: {
     enabled: true,
-    client: "geth" // Can be ganache-cli, geth or parity (default: geth)
+    client: "geth", // Can be ganache-cli, geth or parity (default: geth)
+    port: "30302" // Ethereum P2P port
```

Comment:
I'd only change the whisper one. We just need to fix the port number being used in Whisper and it should work. That way, we keep the default port for the blockchain, since people are used to that port number